Panels/Panels.module.less

```diff
@@ -85,7 +85,6 @@
 
 			.viewport {
 				&.transitioning {
-					-webkit-clip-path: polygon(@agate-panels-breadcrumb-width 0, 100% 0, 100% 100%, @agate-panels-breadcrumb-width 100%);
```

Comment:
There are 2 places more in this file in develop branch, `-webkit-margin-start` and `-webkit-margin-end`. Please fix those properties.

Reply:
OK